src/ncp/ncp_base.hpp

```diff
@@ -559,7 +568,11 @@ class NcpBase
 
     uint8_t mTxBuffer[kTxBufferSize];
 
+#if OPENTHREAD_CONFIG_MULTIPAN_RCP_ENABLE
+    spinel_tid_t mNextExpectedTid[4];
+#else
     spinel_tid_t mNextExpectedTid;
+#endif
 
```

Comment:
Should we get rid of the switch here and just always use the array?

The value 4 should be replaced by (SPINEL_HEADER_IID_MAX + 1) defined in spinel.h.  I will fix that.